stylelint-polaris/configs/coverage.js

```diff
@@ -14,14 +14,18 @@ module.exports = {
     'declaration-property-value-disallowed-list': [
       {
         display: ['grid', 'flex'],
+        // No numeric values
```

Comment:
These are currently disallowing Scss vars: https://regex101.com/r/p6ywGC/1

Reply:
I think we need to update all the regexes here to only pass on polaris custom properties.